tests/RuntimeTest.php

```diff
@@ -27,101 +27,97 @@ protected function setUp(): void
     }
 
     /**
-     * @todo Now that this component is PHP 7-only and uses return type declarations
-     * this test makes even less sense than before
+     * @requires extension xdebug
      */
-    public function testAbilityToCollectCodeCoverageCanBeAssessed(): void
+    public function testCanCollectCodeCoverageWhenXdebugExtensionIsEnabled(): void
     {
-        $this->assertIsBool($this->env->canCollectCodeCoverage());
+        $this->assertTrue($this->env->canCollectCodeCoverage());
     }
 
     /**
-     * @todo Now that this component is PHP 7-only and uses return type declarations
-     * this test makes even less sense than before
+     * @requires extension pcov
      */
+    public function testCanCollectCodeCoverageWhenPcovExtensionIsEnabled(): void
+    {
+        $this->assertTrue($this->env->canCollectCodeCoverage());
+    }
+
+    /**
+     * @requires function phpdbg_break_file
```

Comment:
Not so sure whether this makes sense - what do you think, @sebastianbergmann?

Reply:
Should work. But will break should PHPDBG remove that function at some point. What about
```php
if (\PHP_SAPI !== 'phpdbg`) {
    $this->markTestSkipped('Requires PHPDBG');
}
```
instead?

Reply:
Sounds great, give me a moment!

Reply:
Adjusted!